src/compiler/crystal/tools/init.cr

```diff
@@ -207,7 +207,7 @@ module Crystal
       end
 
       def module_name
-        config.name.split('-').map(&.camelcase).join("::")
+        config.name.gsub(/-([0-9]+)/, "\\1").split('-').map(&.camelcase).join("::")
```

Comment:
```suggestion
        config.name.split('-').compact_map { |name| name.camelcase if name[0]?.try(&.ascii_letter?) }.join("::")
```

Reply:
Thank you for suggestion!
It returns `Crystal::Proj`, so `1` is ignored. Is it OK?

Reply:
You're right, I've updated the suggestion.